docs/understanding/pipeline.rst

```diff
@@ -0,0 +1,30 @@
+.. include:: ../links.inc
+
+.. _pipeline:
+
+The Junifer Pipeline
+====================
+
+The junifer pipeline is the main execution path of junifer. It consists of five steps:
+
+1. :ref:`Data Grabber <datagrabber>`: Interpret the database and provide a list of files.
```

Comment:
`dataset` instead of `database`?

Reply:
It can be either. Given that we are coupled with datalad, I would keep it as dataset.